src/v1/texture.rs

```diff
@@ -15,64 +15,64 @@ enum_number!(
         NearestMipmapLinear = 9986,
         LinearMipmapLinear = 9987,
     }
-);
+}
 
 impl Default for Filter {
     fn default() -> Filter {
         Filter::Nearest
     }
 }
 
-enum_number!(
+enum_number! {
     Wrap {
         Repeat = 10497,
         ClampToEdge = 33071,
         MirroredRepeat = 33648,
     }
-);
+}
 
 impl Default for Wrap {
     fn default() -> Wrap {
         Wrap::Repeat
     }
 }
 
-enum_number!(
+enum_number! {
     Format {
         Alpha = 6406,
         Rgb = 6407,
         Rgba = 6408,
         Luminance = 6409,
         LuminanceAlpha = 6410,
     }
-);
+}
 
 impl Default for Format {
     fn default() -> Format {
         Format::Rgba
     }
 }
 
-enum_number!(
+enum_number! {
     Target {
         Texture2d = 3553,
     }
-);
+}
 
 impl Default for Target {
     fn default() -> Target {
         Target::Texture2d
     }
 }
 
-enum_number!(
+enum_number! {
     TexelType {
```

Comment:
They're not really all upper case. It just looks that way because every 'word' only has one letter. The _ pattern is definitely unconventional, but it conveys meaning much easier. Compare this to [vulkano](https://github.com/tomaka/vulkano/blob/master/vulkano/src/format.rs#L364) and [glium](https://github.com/tomaka/glium/blob/master/src/image_format.rs#L74) and hopefully you'll see what I mean.